### DISCUSSION TITLE
Overlapping both(top/bottom) panels with full screen videos

### DISCUSSION THREAD

**Discussion by aposteeltje-bwg** [nwg-piotr/nwg-shell#244] [Q&A] [answered]
Hi all. Lectori Salutem.

I've installed latest nwg-iso (2024) and, if it means something: i use Desktop-style 'hyperland-0', which has two panels:
a top- and bottom panel.

Now, when i watch a full screen video, eihter via firefox or installed app(smplayer, vlc, etc) .,
the bottom panel gets overlapped, but the top panel stays visible and on top of the full screen video.
As far as i can see, both panels have the the exact same settings.

So, is there a way to also cover/overlap the top panel, when watching full screen videos?

Thanks in advance.
Greets, aposteeltje-bwg

**Comment by nwg-piotr** [comment, marked as answer]
Hi,

In default settings for the hyprland-0 profile, the top panel there is on the overlay panel, and the bottom panel - on the top layer. That's why the top panel covers the fullscreen window. You may change it in the panel settings:

![image](https://github.com/nwg-piotr/nwg-shell/assets/20579136/9009c6aa-1300-4604-bffd-a5326b2b8af3)

Personally I use the "Hide/show signal" to hide both panels while watching videos:

![image](https://github.com/nwg-piotr/nwg-shell/assets/20579136/69370f22-f470-44d4-b0a9-3134549c5216)

Then you need to bind the `pkill --signal 64 nwg-panel` command to some key.

**Reply by aposteeltje-bwg** [reply]
Ah, my bet..
I didn't saw that. 

Thank you btw. for this beautiful project you've made and shared.
It runs perfectly under Nouveau; no need for propr. Nvidia-drivers. :-)

just curious., is there also a way to make the panels (semi)transparent?

**Reply by nwg-piotr** [reply]
Sure. Find a corresponding .css file (in this case: `~/.config/nwg-panel/hyprland-0.css`), then find the "name" used by the panel (actually it should be called CSS ID):

![image](https://github.com/nwg-piotr/nwg-shell/assets/20579136/5416841e-8c55-469a-b81e-44ac3055a4c7)

Then give it a color and transparency of your choice, e.g.:

```css
/* Top panel in sample config uses this name */
#panel-top {
    border-radius: 0;
    background-color: rgba(0, 0, 0, 0.5);
    color: #eeeeee
}
```

And restart Hyprland (`shift` + `Super` + `C`).

![image](https://github.com/nwg-piotr/nwg-shell/assets/20579136/6fa92f0e-67c1-4da9-a1e8-3ef176e4e821)
